### ISSUE TITLE
Running mean and variance

### ISSUE THREAD

**Issue by mgsalem** [taki0112/Switchable_Normalization-Tensorflow#2]
You don't calculate running mean and variance for the batch normalization part.
